gdb-multi-arch/meta.yaml

```diff
@@ -51,7 +50,7 @@ requirements:
 
 test:
   commands:
-    - gdb -ex "print 1" --batch
+    - multi-arch-gdb -ex "print 1" --batch
```

Comment:
> I took the opportunity to rename the output binaries to have the
multi-arch- prefix, because it's really confusing to me to have gdb
not be native gdb when in a conda environment with this package
installed.

I don't mind it. I've been confused too at times.

We'll need to update this doc:
https://docs.memfault.com/docs/mcu/coredump-elf-with-gdb#run-gdb

And probably some code in the backend/app too?

Reply:
yeah, we can probably keep this change out for now, it's separate from the 13.1 update :+1: